test/test_helper.rb

```diff
@@ -1,39 +1,71 @@
 require 'simplecov_setup'
 require 'minitest/autorun'
 require 'quality-measure-engine'
-require 'pry-nav'
 
+Mongo::Logger.logger.level = Logger::WARN
 Mongoid.load!(File.join(File.dirname(__FILE__),"mongoid.yml"), :test)
 
 class MiniTest::Unit::TestCase
 
   def load_system_js
-    Mongoid.default_session['system.js'].find.remove_all
+    Mongoid.default_client['system.js'].delete_many({})
     Dir.glob(File.join(File.dirname(__FILE__), 'fixtures', "library_functions", '*.js')).each do |json_fixture_file|
       name = File.basename(json_fixture_file,".*")
       fn = "function () {\n #{File.read(json_fixture_file)} \n }"
-      Mongoid.default_session['system.js'].find('_id' => name).upsert(
+      Mongoid.default_client['system.js'].update_one({
+          "_id" => name},
         {
           "_id" => name,
           "value" => BSON::Code.new(fn)
-        }
+        },{upsert: true}
       )
     end
 
   end
 
-  # Add more helper methods to be used by all tests here...
 
-  def collection_fixtures(db, collection, *id_attributes)
```

Comment:
Since the new `collection_fixtures` function doesn't deal with `id_attributes` anymore, we should probably go through the calls to it and get rid of the `_id` strings that get passed into it. there's one in `map_reduce_executor_test.rb` and one in `measure_calculation_job_test.rb`.